src/components/Text/Text.tsx

```diff
@@ -7,22 +7,27 @@ type TextProps = {
   children: string;
   className?: string;
   title?: string;
+  titleAs?: 'h3' | 'p';
   variant?: 'text' | 'text-snippet';
 };
 
 export const Text = ({
   children,
   className,
   title,
+  titleAs = 'h3',
   variant = 'text'
 }: TextProps) => {
+  const TitleElement = variant === 'text-snippet' ? 'p' : titleAs;
```

Comment:
As discussed, remove titleAs for now and make the ternary `const TitleElement = variant === 'text-snippet' ? 'p' : 'h3';`

Reply:
As a title is passed only for `text-snippet` variant I just changed the title to be `p`.